### DISCUSSION TITLE
Fairy正在检索信息中...

### DISCUSSION THREAD

**Discussion by AIFairy** [share121/inter-knot#881] [其他]
**提问：这还是新艾利都得绳网吗？**
![QQ20240810-180027](https://github.com/user-attachments/assets/6eceb5b5-f1a3-497c-b6f6-aedbafd945dd)

**Comment by YTYX-Github** [comment]
回答：逝    的

**Comment by SixRadium** [comment]
这里是旧艾利都（确信）

**Comment by CocoonedSoul** [comment]
大抵应该是的，不过我们这可没有空洞

**Comment by sky-R6** [comment]
是的
但是我還有一個問題
聽聞你的耗電量驚人，這是真的嗎
Fairy 請回答一下

**Comment by CrIsOwO6** [comment]
是的，世界全剧终，欢迎来到新艾利都

**Comment by blqdswkl** [comment]
登！得得登！

**Comment by FrostPhoenix79** [comment]
回答:是的  („ಡωಡ„)

**Comment by 1451374815** [comment]
是的

**Comment by yileinafs** [comment]
回答:是的

**Comment by adrianlao647** [comment]
旧艾莉都欢迎你

**Comment by 12344856** [comment]
嗯呢嗯呢<img src="https://bbs-static.miyoushe.com/static/2023/04/28/d7dd7cd39d84b9964a13f157f09ed7a0_4748799527738820824.png" alt="米仔_探头" width="75" >

**Comment by zhuongxt** [comment]
yes

**Comment by Bugamun** [comment]
自然。

**Comment by lyx1662** [comment]
是的！

**Comment by Cherss111** [comment]
嗯呢

**Comment by yindiea** [comment]
当然，所以你可以发布一些委托吗？

**Comment by Enigma-Soul** [comment]
嗯呢

**Comment by Momsaysnamesthataretoolongmakefoolslook** [comment]
是的，但是连接上别的世界的网络了

**Comment by loadinng** [comment]
肯定
